Content.Server/Sticky/Systems/StickySystem.cs

```diff
@@ -138,6 +150,15 @@ private void OnUnstickSuccessful(UnstickSuccessfulEvent ev)
             return;
 
         UnstickFromEntity(ev.Uid, ev.User, component);
+
+
+        if(EntityManager.HasComponent<IsStuckOnEntityComponent>(ev.Uid))
+            EntityManager.RemoveComponent<IsStuckOnEntityComponent>(ev.Uid);
+        else { Log.Warning("stuck-on-entity-without-stuck-comp");}
+
+        if(EntityManager.HasComponent<HasEntityStuckOnComponent>(ev.Target))
+            EntityManager.RemoveComponent<HasEntityStuckOnComponent>(ev.Target);
+        else { Log.Warning("has-entity-stuck-on-without-stuck-comp");}
```

Comment:
```suggestion
        if (EntityManager.HasComponent<IsStuckOnEntityComponent>(ev.Uid))
            EntityManager.RemoveComponent<IsStuckOnEntityComponent>(ev.Uid);
        else
            Log.Warning("stuck-on-entity-without-stuck-comp");

        if (EntityManager.HasComponent<HasEntityStuckOnComponent>(ev.Target))
            EntityManager.RemoveComponent<HasEntityStuckOnComponent>(ev.Target);
        else
            Log.Warning("has-entity-stuck-on-without-stuck-comp");
```
Minor formatting stuff.